tasks/knative.py

```diff
@@ -0,0 +1,286 @@
+import os
+from os.path import join
+from subprocess import call, check_output
+
+from invoke import task
+
+FAASM_VERSION = "0.5.9"
```

Comment:
If we did have Faasm as a submodule of this project we could read this version from the `VERSION` file in the submodule. The Faasm version is currently written in a couple of places.